src/domain.types/statistics/daily.statistics/daily.statistics.dto.ts

```diff
@@ -0,0 +1,16 @@
+export interface YearWiseUsers{
+    Year?:number;
+    UserCount?:number;
+}
+
+export interface YearWiseDeviceDetails{
+    Year?:number;
```

Comment:
Applicable everywhere - 
Proper spacing is needed around colon. 
Alignment is preferred.